doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -91,36 +91,41 @@ Numerals
      sign    : + | -
 
 Strings
-  Strings are delimited by ``"`` (double quote), and enclose a sequence of
-  any characters different from ``"`` or the sequence ``""`` to denote the
-  double quote character. In grammars, the entry for quoted strings is
-  :token:`string`.
-
-  .. productionlist:: coq
-     string : " `characters_opt` "
-     characters_opt : characters_opt character
-                    : empty
+  Strings begin and end with ``"`` (double quote).  Use ``""`` to represent
+  a double quote character within a string.  In the grammar, strings are
+  identified with :production:`string`.
 
 Keywords
-  The following identifiers are reserved, and cannot be used as identifiers::
+  The following character sequences are reserved keywords that cannot be
+  used as identifiers::
 
-    _ as at cofix else end exists exists2 fix for
-    forall fun if IF in let match return
-    SProp Prop Set Type then using where with
+    _ Axiom CoFixpoint Definition Fixpoint Hypothesis IF Parameter Prop
+    SProp Set Theorem Type Variable as at by cofix discriminated else
+    end exists exists2 fix for forall fun if in lazymatch let match
+    multimatch return then using where with
+
+  Note that plugins may define additional keywords when they are loaded.
 
 Special tokens
-  The following sequences of characters are special tokens::
-
-    ! #[ % & && ( () ) * + ++ , - -> . .( ..
-    / /\ : :: :< := :> ; < <- <-> <: <= <> =
-    => =_D > >-> >= ? ?= @ [ \/ ] ^ { | |-
-    || } ~
-
-  Lexical ambiguities are resolved according to the “longest match”
-  rule: when a sequence of non alphanumerical characters can be
-  decomposed into several different ways, then the first token is the
-  longest possible one (among all tokens defined at this moment), and so
-  on.
+  The set of special
+  tokens defined at any given time can vary because the :cmd:`Notation`
+  command can define new tokens.  A :cmd:`Require` command may load more notation definitions,
+  while the end of a :cmd:`Section` may remove notations.
+
+  Here are some of the character sequences that are special tokens::
+
+    ! #[ % & ' ( () (bfs) (dfs) ) * ** + , - ->
+    . .( .. ... / : ::= := :> :>> ; < <+ <- <:
+    <<: <= = => > >-> >= ? @ @{ [ [= ] _ _eqn
+    `( `{ { {| | |- || }
+
+
+  When multiple tokens match the beginning of a sequence of characters,
+  the longest matching token is used.
+  Occasionally you may need to insert spaces to separate tokens.  For example,
+  if ``~`` and ``~~`` are both defined as special tokens, the inputs ``~ ~`` and
+  ``~~`` generate different tokens, whereas if `~~` is not defined, then the
```

Comment:
whereas if `~~` were not defined

Reply:
I think it is correct as-is:
- It is a "zero conditional" expressing a factual implication (time independent), which usually uses the indicative rather than the subjunctive:
https://en.wikipedia.org/wiki/English_conditional_sentences#Zero_conditional
- If I used "were", I would have to use "would be equivalent".  Wordy.
- "were" is a past form while the rest of the sentence is in the present tense.  If "were" is needed here, it would also be needed in the "if ``~`` and ``~~`` are both defined" as well as "would generate"